components/bash.yml

```diff
@@ -3,4 +3,6 @@ packages:
 - bash
 rules:
 - accounts_umask_etc_bashrc
+- accounts_umask_etc_csh_cshrc
```

Comment:
Not strictly bash, but only have one rule in the csh component also seems odd.

I think I'm okay with this for now.